src/main/java/co/omise/models/SourceType.java

```diff
@@ -33,8 +33,10 @@ public enum SourceType {
     InstallmentKtc,
     @JsonProperty("installment_scb")
     InstallmentScb,
-    @JsonProperty("installment_citi")
```

Comment:
do we add tests to SourceTypeTest also?

Reply:
added tests here https://github.com/omise/omise-java/pull/143/commits/76fdeae86ce492459e2602620b303f720270367e